specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2019-08-01/cosmos-db.json

```diff
@@ -3689,6 +4223,45 @@
       },
       "description": "The List operation response, that contains the containers and their properties."
     },
+    "SqlStoredProcedureListResult": {
+      "properties": {
+        "value": {
```

Comment:
>value [](start = 9, length = 5)

Additional comment about paging. You should declare the nextLink property in the swagger as well.

Reply:
Hey, I am planning to take this up in a subsequent PR, for all the resources, same for the default error response.